connectors/source-aws-billing/README.md

```diff
@@ -87,8 +87,16 @@ docker run -it --rm --network=host \
 ```
 
 ### Test
+<<<<<<< HEAD
+=======
+<<<<<<< HEAD
 
 Before starting Amazon billing Source use the following command to run a Display sink, which will receive and prints the incoming CloudEvents.
+=======
+>>>>>>> main
+**To test the connector, run the Display Sink first before running AWS Billing source.**
+Open a terminal and use the following command to run a Display sink, which receives and prints CloudEvents.
```

Comment:
compare with [line 94](https://github.com/linkall-labs/vanus-connect/pull/210/files#diff-289f0dc5100fa339c3bf3ac322734dd5e3520c0782685a3df211ef0289ad5a20R94) and merge them